cpp/benchmarks/transpose/transpose.cpp

```diff
@@ -42,7 +44,7 @@ static void BM_transpose(benchmark::State& state)
   }
 
   // Collect memory statistics.
-  auto const bytes_read    = input.num_columns() * input.num_rows() * (sizeof(int32_t));
+  auto const bytes_read    = input.num_columns() * input.num_rows() * cudf::size_of(column_type);
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**💡 suggestion:** ‏ This is one way to do it. But I think the way I suggested is a bit better because it all happens at compile time, whereas `cudf::size_of()` invokes the type dispatcher at run time. Not that it will affect benchmarks, but it just seems cleaner to use `sizeof(cudf::id_to_type<column_type>)`.